client/components/auth/loginRegister-modal.js

```diff
@@ -0,0 +1,40 @@
+import React, { Component } from 'react';
+import Modal from 'react-bootstrap/lib/Modal';
+import SigninForm from './signin-form';
+import SignupForm from './signup-form';
+import { Tabs, TabLink, TabContent } from 'react-tabs-redux';
+import { SIGNIN_MODAL } from '../../constants';
+import { MODAL_CLOSE } from '../../actions/action-types';
+require('../style/_header.scss');
+
+export default class SigninModal extends Component {
```

Comment:
here, I think you forget to rename `SigninModal` to something like `LoginRegisterModal`...